### ISSUE TITLE
Search doesn't allow to search for anything other than Tracks

### ISSUE THREAD

**Issue by morphis** [Elleo/cutespotify#18]
The combobox doesn't allow choosing anything else. Trying to show available options shows nothing and "Tracks" is selected by default.

**Comment by shdpl** [comment]
You need to:
* tap "Tracks"  so icon on the right will be a tick ( instead of arrow down )
* swipe from "Tracks" to top until you'll find option you need

I don't know whether they changed this control or what. It was very non-obvious to me too.

**Comment by dinilu** [comment]
This is true for the version in Ubuntu Store (1.4.3) but it does not work for the version in Open Store (1.6.1). At least in a BQ Aquaris M10 FHD Ubuntu Edition. Anyway, it would benice if the combobox expanded and showed all options.
